spec.bs

```diff
@@ -1,21 +1,10 @@
 <pre class="metadata">
-Title: URLPattern API
+Group: WHATWG
+H1: URL Pattern
 Shortname: urlpattern
-Repository: WICG/urlpattern
-Inline Github Issues: true
-Group: WICG
-Status: CG-DRAFT
-Level: 1
-URL: https://wicg.github.io/urlpattern/
-Boilerplate: omit conformance, omit feedback-header
-Editor: Ben Kelly, Google https://www.google.com/, wanderview@chromium.org
-Abstract: The URLPattern API provides a web platform primitive for matching URLs based on a convenient pattern syntax.
-Status Text: This proposal will hopefully move into the WHATWG URL workstream in the future.
-!Participate: <a href="https://github.com/WICG/urlpattern">GitHub WICG/urlpattern</a> (<a href="https://github.com/WICG/urlpattern/issues/new">new issue</a>, <a href="https://github.com/WICG/urlpattern/issues?state=open">open issues</a>)
-!Commits: <a href="https://github.com/WICG/urlpattern/commits/main/spec.bs">GitHub spec.bs commits</a>
-Complain About: accidental-2119 yes, missing-example-ids yes
+Text Macro: TWITTER urlpatternapi
+Abstract: The URL Pattern Standard provides a web platform primitive for matching URLs based on a convenient pattern syntax.
 Indent: 2
```

Comment:
We'll update spec_factory to allow 2-space indents here instead of the usual 1-space.